src/locales/en/common.json

```diff
@@ -382,7 +383,9 @@
     "privateKeyRevealedPurpose": "Please use this private key in order to restore the wallet.",
     "doNotShareBackup": "Keep it secure as it's the only way to recover your account in an emergency. Don't email or screenshot it.",
     "sessionExpiredReEnterPin": "Login session expired, please enter your PIN to proceed.",
-    "addContacts": "Add contacts to address book to easily access favorite addresses for sending assets."
+    "addContacts": "Add contacts to address book to easily access favorite addresses for sending assets.",
+    "cautionMessage": "All chains use single wallet address.\n Receive <0>{{ethereum}}</0> assets",
+    "withCaution": "with caution"
```

Comment:
When reading whole sentence, it feels like it lacks a dot at the end: "Receive Ethereum assets with caution"

Reply:
Done.